prow/config/config.go

```diff
@@ -70,12 +70,33 @@ type Config struct {
 
 	// PushGateway is a prometheus push gateway.
 	PushGateway PushGateway `json:"push_gateway,omitempty"`
+
+	// Owners contains configuration related to handling OWNERS files.
+	Owners Owners `json:"owners,omitempty"`
```

Comment:
Are we going to use OWNERS for something other than prow plugins?

Reply:
I scoped the OWNERS client so that non plugin components can use it if needed. Currently there are no plans to need this, but I was thinking that alias resolution and file ownership mapping could be useful outside of plugins in the future.
I'm happy to move this to the plugin scope though if that is preferable.